### ISSUE TITLE
Fix sinon-test depcrecation

### ISSUE THREAD

**Issue by Coteh** [Coteh/mcu-rewatch#23]
```
'sinonTest.configureTest is deprecated and will be removed from the public API in a future version of sinon-test'
```

Replace all calls to `sinonTest.configureTest` to simply `sinonTest`.

**Comment by Coteh** [comment]
Until [this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36558) is merged into DefinitelyTyped, the following TypeScript error will occur:
```
Cannot invoke an expression whose type lacks a call signature. Type 'sinonTest' has no compatible call signatures.
```
This is because `sinonTest` is currently defined as an interface with two methods, but has since been updated to be a function itself with an additional property defined within it for the deprecated `configureTest` method. The PR linked above will fix this.

**Comment by Coteh** [comment]
Fixed with #25
